kubernetes/cray-ims/templates/post_upgrade_hook.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: ims-post-upgrade-hook1-container
-        image: "baseos/alpine:3.13"
+        image: "arti.dev.cray.com/baseos-docker-master-local/alpine:3.13"
```

Comment:
Is this the correct image path?  Zach said arti.dev.cray.com would be deprecated in 1.2: https://cray.slack.com/archives/C02H5FEPHBK/p1636501974195300?thread_ts=1636501736.195200&cid=C02H5FEPHBK

Reply:
Updated to use artifactory.algol60.net/docker.io/alpine:latest